instagramapp/src/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import './App.css';
+import dummyData from './dummy-data';
+import SearchBar from './components/SearchBar/SearchBar'
+import PostContainer from './components/PostContainer/PostContainer'
+
+
+
+// The root App component of your application should import the dummy data from the dummy-data.js file with import dummyData from './dummy-data'; and iterate over said data, passing each individual object as a prop to an instance of PostContainer.
```

Comment:
its really great to see you using comments here, its super helpful for organization and helps you and others understand what your code is doing better.

always a good idea to write out how your application should be working before implementing it